src/backend/tools.ts

```diff
@@ -212,123 +217,93 @@ export const DXVK = {
+              prefix: LogPrefix.DXVKInstaller
+            })
+          }
         })
-        resolve(true)
       })
       return true
     }
 
     logInfo([`installing ${tool} on...`, prefix], {
       prefix: LogPrefix.DXVKInstaller
     })
-    // backup current dlls on the prefix
+
     if (currentVersion === globalVersion) {
       logInfo(`${tool} already installed!`, {
         prefix: LogPrefix.DXVKInstaller
       })
       return true
     }
 
-    logInfo(`Backing up current ${tool} dlls`, {
-      prefix: LogPrefix.DXVKInstaller
-    })
-
-    await new Promise((resolve) => {
-      dlls.forEach(async (dll) => {
-        const dllPath = `${winePrefix}/drive_c/windows/system32/${dll}`
-        if (existsSync(dllPath)) {
-          renameSync(
-            `${winePrefix}/drive_c/windows/system32/${dll}`,
-            `${winePrefix}/drive_c/windows/system32/${dll}.bak`
-          )
-        }
-
-        // macOs supports 64 bits only
-        if (!isMac) {
-          const dllPath64 = `${winePrefix}/drive_c/windows/syswow64/${dll}`
-          if (existsSync(dllPath64)) {
-            renameSync(
-              `${winePrefix}/drive_c/windows/syswow64/${dll}`,
-              `${winePrefix}/drive_c/windows/syswow64/${dll}.bak`
-            )
+    // copy the new dlls to the prefix
+    dlls.forEach((dll) => {
+      if (!isMac) {
+        copyFile(
+          `${toolPathx32}/${dll}`,
+          `${winePrefix}/drive_c/windows/syswow64/${dll}`,
+          (err) => {
+            if (err) {
+              logError([`Error when copying ${dll}`, err], {
+                prefix: LogPrefix.DXVKInstaller
+              })
+            }
           }
-        }
-      })
-      resolve(true)
-    })
+        )
+      }
 
-    // copy the new dlls to the prefix
-    await new Promise((resolve) => {
-      dlls.forEach((dll) => {
-        if (!isMac) {
-          copyFileSync(
-            `${toolPathx32}/${dll}`,
-            `${winePrefix}/drive_c/windows/syswow64/${dll}`
-          )
+      copyFile(
+        `${toolPathx64}/${dll}`,
+        `${winePrefix}/drive_c/windows/system32/${dll}`,
+        (err) => {
+          if (err) {
+            logError([`Error when copying ${dll}`, err], {
+              prefix: LogPrefix.DXVKInstaller
+            })
+          }
         }
-
-        copyFileSync(
-          `${toolPathx64}/${dll}`,
-          `${winePrefix}/drive_c/windows/system32/${dll}`
-        )
-      })
-      resolve(true)
+      )
     })
 
     // register dlls on the wine prefix
-    await new Promise((resolve) => {
-      dlls.forEach(async (dll) => {
-        await execAsync(
-          `WINEPREFIX='${winePrefix}' '${wineBin}' reg add 'HKEY_CURRENT_USER\\Software\\Wine\\DllOverrides' /v ${dll} /d native /f `,
-          execOptions
-        )
-          .then(() => {
-            logInfo(`${dll} registered!`, {
+    const overrideType = isMac ? 'native,builtin' : 'native'
```

Comment:
`native,builtin` is the "default" DLL override as far as I know, so DLL overrides shouldn't be necessary at all on MacOS

I still wonder if they could also be skipped on Linux, *in theory* they shouldn't do anything (since the default load order is "native,builtin" as mentioned above, and a `native` DLL will now always be found)

Reply:
Still, it's not causing any harm adding this here.

The point is. It is native only if you have both dlls for 32 and 64 bits. Native built-in it's when yiu have only one of them.

I think it's no harm to keep this until we proper check on both platforms that it's fine

Reply:
Ok, removed then.